### PR TITLE
Removed latest post from all blog and added related posts to single blog.

### LINKED ISSUE #1125: [NG] latest blogs section
**Current Behavior**
The sidebar of the https://layer5ng.netlify.app/blog contains a `Latest Post` section. It is not populated currently

**Desired Behavior**
Populate the section with 4 latest blog post. 
Note - This is to be done dynamically using graphql query.  

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by OmkarPh**
**Description**
All blogs page has only sidebar without redundant latest posts.
Single blogs have only related posts section at bottom (Fetches latest posts for now, change query later. )

This PR fixes #1125 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Blog-single/index.js [hunk 1]
```diff
@@ -56,11 +56,9 @@ const BlogSingle = ({data}) => {
                                 </Col>
                             </Row>
                         </Col>
-                        <Col sm={12} lg={4}>
-                            <Sidebar tags={allTags.allMdx.group}/>
-                        </Col>
                     </Row>
                 </Container>
+                <RelatedPosts/>
```
Comment (Tanuj22) [inline src/sections/Blog-single/index.js:61]: Move this inside container

File: src/sections/Related-Posts/relatedPosts.style.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
+import styled from "styled-components";
+
+
+const RelatedPostsWrapper = styled.div`
+        .search-box {
```
Comment (Tanuj22) [inline src/sections/Related-Posts/relatedPosts.style.js:5]: These classes are not needed right?

**Review by Tanuj22 — COMMENTED** [review]
Are we not keeping anything on the sidebar for single blog post :thinking: @Nikhil-Ladha. I not, the blog should be in the center

**Comment by OmkarPh** [comment]
@Nikhil-Ladha suggested me to remove sidebar from single blog, so I did it ...

File: src/sections/Related-Posts/relatedPosts.style.js (hunk 2 above)
Comment (OmkarPh) [reply src/sections/Related-Posts/relatedPosts.style.js:5]: Ya, I forgot to amend that file 🤦🏻‍♂️

File: src/sections/Blog-single/index.js (hunk 1 above)
Comment (OmkarPh) [reply src/sections/Blog-single/index.js:61]: Ok

**Comment by Tanuj22** [comment]
> @Nikhil-Ladha suggested me to remove sidebar from single blog, so I did it ...

Okay. The blog should be in the center of the page then

**Comment by Tanuj22** [comment]
Also please fix the DCO

**Comment by OmkarPh** [comment]
> Also please fix the DCO

Can you tell me how to amend previously signed commits?

**Comment by Nikhil-Ladha** [comment]
Follow this link https://github.com/layer5io/layer5/pull/1175/checks?check_run_id=1435467111

**Comment by OmkarPh** [comment]
> Follow this link https://github.com/layer5io/layer5/pull/1175/checks?check_run_id=1435467111

Why do i get the above error...

**Comment by Nikhil-Ladha** [comment]
Check if you are on the correct branch.

**Comment by OmkarPh** [comment]
> Check if you are on the correct branch.

Ya, I'm on that branch, but the thing is, I signed last commit but last second one is unsigned,
image

**Comment by kishore007k** [comment]
@OmkarPh DO the following to fix the DCO error
- `git checkout <branchname>`
- `git commit -s --amend`
- `git push --force origin <branchname>`

**Comment by OmkarPh** [comment]
> @OmkarPh DO the following to fix the DCO error
> 
> * `git checkout <branchname>`
> * `git commit -s --amend`
> * `git push --force origin <branchname>`

I'm getting same error again :( on `git commit -s --amend`

The unsigned commit is last second one and not the last one, maybe that is the issue...

**Comment by kishore007k** [comment]
> > @OmkarPh DO the following to fix the DCO error
> > 
> > * `git checkout <branchname>`
> > * `git commit -s --amend`
> > * `git push --force origin <branchname>`
> 
> I'm getting same error again :( on `git commit -s --amend`
> 
> The unsigned commit is last second one and not the last one, maybe that is the issue...

Are you sure? But It seems to me in the error is from the last commit.

**Comment by OmkarPh** [comment]
> > Check if you are on the correct branch.
> 
> Ya, I'm on that branch, but the thing is, I signed last commit but last second one is unsigned,
> image

As you can see in this Screenshot, last seccond one was unsigned

**Comment by Nikhil-Ladha** [comment]
No worries, DCO can sometimes be problematic. Let's skip that 🙂

**Comment by OmkarPh** [comment]
> No worries, DCO can sometimes be problematic. Let's skip that 🙂

Pardon me, I will config my setup to signoff all commits by default, so this doesn't happen again :)

**Comment by OmkarPh** [comment]
> > @Nikhil-Ladha suggested me to remove sidebar from single blog, so I did it ...
> 
> Okay. The blog should be in the center of the page then

I increased width of blog, is it OK? See this:
[Performance benchmarking using Meshery and Nighthawk _ Layer5 - The Service Mesh Company.pdf](https://github.com/layer5io/layer5/files/5579068/Performance.benchmarking.using.Meshery.and.Nighthawk._.Layer5.-.The.Service.Mesh.Company.pdf)

**Comment by Nikhil-Ladha** [comment]
Just one thing, in mobile view.
For single blog page, the screen is spilling on the right. I assume that that can be fixed by wrapping the `RelatedPost` component within the ` Row` or `Col` component. Right?

**Comment by OmkarPh** [comment]
> Just one thing, in mobile view.
> For single blog page, the screen is spilling on the right. I assume that that can be fixed by wrapping the `RelatedPost` component within the ` Row` or `Col` component. Right?

I didn't get that..
Here is SS from my mobile:
Image 2020-11-22 at 12 08 31 PM

**Comment by Nikhil-Ladha** [comment]
For the single blogs page, the content is not center aligned.
Current view:
Screenshot from 2020-11-22 12-32-49
Screenshot from 2020-11-22 12-33-01

Desired view:
Screenshot from 2020-11-22 12-33-28
Screenshot from 2020-11-22 12-33-35

>I didn't get that..
Look at the screenshot below. There is a horizontal scrollbar being introduced in the mobile view due to the spilling of white space on the right. This generally occurs due to extra padding/margin.

Screenshot from 2020-11-22 12-41-25

Notice the white space beside the footer, and it is caused due to the padding of `-15px` on the row, check how is that introduced and why is it there, removing that should fix the problem.

**Comment by OmkarPh** [comment]
Hi @Nikhil-Ladha 
I tried so hard, but couldn't reproduce the horizontal scroll issue you were talking about :(
I used exact same responsive resolution as yours:
image

I tried in both Opera and Chrome browsers, but couldn't reproduce that white spill, 
I made one more commit, can you check that out, maybe you saw issues before that commit was pushed.

**Comment by OmkarPh** [comment]
And about centred blog,
Can you check deploy once again, it would have changed after my last commit, it is centered for me
image

**Comment by Nikhil-Ladha** [comment]
Yup, interestingly that did solved the issue 🤔

File: src/sections/Blog-single/index.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import slugify from "../../utils/slugify";
 
 import { Container, Row, Col } from "../../reusecore/Layout";
 import PageHeader from "../../reusecore/PageHeader";
-import Sidebar from "../Blog-sidebar";
+import RelatedPosts from "../Related-Posts";
 
 import BlogPageWrapper from "./blogSingle.style";
```
Comment (Nikhil-Ladha) [inline src/sections/Blog-single/index.js:12]: It seems like this page doesn't require a graphql query anymore, right?
@OmkarPh , @Tanuj22

Comment (Tanuj22) [reply src/sections/Blog-single/index.js:12]: Yes it can be removed

Comment (Tanuj22) [reply src/sections/Blog-single/index.js:12]: Yes it can be removed

Comment (OmkarPh) [reply src/sections/Blog-single/index.js:12]: I don't know what that query in Blog-single does, so I didn't touch it :)

Comment (OmkarPh) [reply src/sections/Blog-single/index.js:12]: Is there anything else to change?

File: src/sections/Related-Posts/relatedPosts.style.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                margin-top: 52px;
+            }
+            ul{
+                margin: 0;
+                padding: 0;
+                list-style: none;
+            }
+            
+            
+            &.catagorie{
```
Comment (Tanuj22) [inline src/sections/Related-Posts/relatedPosts.style.js:41]: These can also be removed. Please remove all necessary classes

File: src/sections/Blog-single/index.js (hunk 3 above)
Comment (Tanuj22) [reply src/sections/Blog-single/index.js:12]: Remove this query

**Comment by Tanuj22** [comment]
@OmkarPh These clean ups and we are good to go :)

File: src/sections/Related-Posts/relatedPosts.style.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            &+.sidebar-widgets{
+                margin-top: 52px;
+            }
+            ul{
+                margin: 0;
+                padding: 0;
+                list-style: none;
+            }
+            
+            &.tags{
```
Comment (Tanuj22) [inline src/sections/Related-Posts/relatedPosts.style.js:40]: Even these can be removed right

Comment (OmkarPh) [reply src/sections/Related-Posts/relatedPosts.style.js:40]: We have list of related posts there, so these tags are effective, aren't they?

Comment (Tanuj22) [reply src/sections/Related-Posts/relatedPosts.style.js:40]: I don't think so. You can try to remove and check if makes any difference

Comment (OmkarPh) [reply src/sections/Related-Posts/relatedPosts.style.js:40]: Can you tell me exactly which classes to check for :)

**Review by Tanuj22 — APPROVED** [review]
LGTM! Thanks for sticking through this till the end :)

**Comment by leecalcote** [comment]
Wonderful

**Comment by OmkarPh** [comment]
> Wonderful. The related posts is a good addition. 

Well, the query still fetches latest posts, so don't forget to change it soon 😁

**Pull request merged**